fc/parsing/CompactSyntaxParser.py

```diff
@@ -204,9 +204,9 @@ def OperatorOperands(self):
                 except StopIteration:
                     break
 
-        def Operator(self, operator):
-            """Get the MathML element for the given operator."""
-            return getattr(M, self.OP_MAP[operator])
+        # def operator(self, operator):
```

Comment:
Can we just delete this, and the definition of `OP_MAP`?

Reply:
ok!